packages/p2p/src/components/my-ads/my-ads-table.jsx

```diff
@@ -61,6 +61,19 @@ const MyAdsTable = () => {
                         />
                     </div>
                 )}
+                <div className='p2p-my-ads__warning'>
+                    <HintBox
+                        icon='IcAlertWarning'
+                        message={
+                            <Text as='p' size='xxxs' color='prominent' line_height='xs'>
+                                <Localize
+                                    i18n_default_text={`Floating rates are enabled for ${general_store.client.local_currency_config.currency}. Ads with fixed rates will be deactivated. Switch to floating rates by`}
+                                />
```

Comment:
Localize doesn't accept ` string operator, please use `values`.

Reply:
Done, thx for notice!